scripts/autolink-ios.rb

```diff
@@ -41,6 +41,7 @@ def use_react_native! (options={})
   pod 'DoubleConversion', :podspec => "#{prefix}/third-party-podspecs/DoubleConversion.podspec"
   pod 'glog', :podspec => "#{prefix}/third-party-podspecs/glog.podspec"
   pod 'Folly', :podspec => "#{prefix}/third-party-podspecs/Folly.podspec"
+  pod 'boost-for-react-native', :podspec => "#{prefix}/third-party-podspecs/boost-for-react-native.podspec"
```

Comment:
Are we using the autolink-ios.rb file for Mac? Probably worth cleaning it up by making an autolink-apple.rb file that has everything shared between ios/mac and then autolink-ios and autolink-mac (you'll have to make this) would pull it in.

Then where we consume the autolink-ios.rb file right now we would just conditionally include the appropriate platform extension version based off the platform.

Reply:
Good question; yes and no.

The “autolinking” that’s being referred to in this version of the Podfile is an earlier incarnation, the current version lives in [react-native-community/cli](https://github.com/react-native-community/cli/blob/master/packages/platform-ios/native_modules.rb). That one will definitely need updating for macOS, but I’ll do that in a follow-up PR [when I make a little RN macOS example app I have in mind].

As-is, the old version in this repo works for macOS too and is simply named `autolinking-ios` rather than actually only working for iOS.